apps/web/components/NavbarLink.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { TouchableWithoutFeedback } from '@kiwicom/margarita-components';
+import { Text, StyleSheet } from '@kiwicom/universal-components';
+import Router from 'next/router';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+type Props = {|
+  +label: string,
+  +route: string,
+|};
+
+type State = {|
+  +isActive: boolean,
+|};
+
+export default class NavbarLink extends React.Component<Props, State> {
+  state = {
+    isActive: false,
+  };
+
+  componentDidMount() {
+    if (Router.route === this.props.route) {
+      this.onMouseEnter();
```

Comment:
Just one think this naming `this.onMouseEnter();` is confusing a little bit.
probably just will be more clearly: 
```
 this.setState({ isActive: true });
```

Reply:
Yes, agreed, however there is an eslint-rule saying `no-set-state` in componentDidMount. 

Ideally, we should have done this in the constructor, however, `next/router` can not be accessed on the server, and constructor is executed also on server.

so then we would need to change it to: 

```
this.setState({ isActive: true }); // eslint-disable-line react/no-did-update-set-state
```